### PR TITLE
Remove unused CRDs and mock objects

### PR DESCRIPTION
**Pull request by baijum**
Signed-off-by: Baiju Muthukadan <baiju.m.mail@gmail.com>

### CODE REVIEW

**Comment by pedjak** [comment]
this files are not used anywhere at all, they are leftovers from some early code versions, and IMHO it is safe to remove them fully instead. /cc @pmacik

**Comment by pmacik** [comment]
@pedjak is correct. Those are a remnance of the past not used any longer and can be deleted.
Additionally the following pieces that used to depend on it can also be removed:
* `test/acceptance/features/steps/etcdoperator.py` file
* `test/acceptance/features/steps/etcdcluster.py` file
* the following step definitions in `test/acceptance/features/steps/steps.py`:
  * `Etcd operator running`
  * `Etcd cluster "{etcd_name}" is running`
* `test/acceptance/features/steps/postgres_db.py` file
* all related references/functions in `test/mocks/mocks.go`
* all related references/functions in `test/mocks/fake.go`

**Comment by baijum** [comment]
Thanks @pedjak !

@pedjak I have removed the unused CRDs and mock objects. Also updated the PR title.

File: test/acceptance/features/steps/steps.py [hunk 1]
```diff
@@ -13,13 +13,10 @@
 
 from behave import given, register_type, then, when, step
 from dboperator import DbOperator
```
Comment (pmacik) [inline test/acceptance/features/steps/steps.py:15]: @baijum This should be also removed along with the class and step:
see the https://github.com/baijum/service-binding-operator/pull/2

**Review by pmacik — CHANGES_REQUESTED** [review]

File: test/acceptance/features/steps/steps.py (hunk 1 above)
Comment (baijum) [reply test/acceptance/features/steps/steps.py]: Thanks for the PR. I have merged it.

**Pull request merged**